### PR TITLE
KOGITO-2771: Cucumber Tests: Cover Webhooks in KogitoBuild

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA: https://issues.redhat.com/browse/KOGITO-2771
Description: Cover that the kogito cloud operator initialises correctly the build config in Openshift.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by Sgitario** [comment]
I ran the new feature build_webhooks.feature successfully.

File: test/framework/kogitobuild.go [hunk 1]
```diff
@@ -60,6 +62,13 @@ func cliDeployKogitoBuild(buildHolder *bddtypes.KogitoBuildHolder) error {
 		cmd = append(cmd, gitURI)
 	}
 
+	// webhooks
+	if len(buildHolder.Spec.WebHooks) > 0 {
+		for _, webhook := range buildHolder.Spec.WebHooks {
+			cmd = append(cmd, "--web-hook", fmt.Sprintf("%s=%s", webhook.Type, webhook.Secret))
```
Comment (sutaakar) [inline test/framework/kogitobuild.go:68]: Can you move the code to the mappers.GetBuildCLIFlags among other KogitoBuild flags?

File: test/features/build_webhooks.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Feature: Configure Webhooks Triggers in Kogito Build
```
Comment (sutaakar) [inline test/features/build_webhooks.feature:1]: Wouldn't it have more sense to put the scenario into https://github.com/kiegroup/kogito-cloud-operator/blob/master/test/features/deploy_kogito_build.feature ?

Comment (Sgitario) [reply test/features/build_webhooks.feature:1]: Moved

File: test/framework/kogitobuild.go (hunk 1 above)
Comment (Sgitario) [reply test/framework/kogitobuild.go:68]: Moved

**Pull request merged**